packages/plugin-credentials/src/index.js

```diff
@@ -0,0 +1,23 @@
+/**!
+ *
+ * Copyright (c) 2015-2016 Cisco Systems, Inc. See LICENSE file.
+ * @private
+ */
+
+import '@ciscospark/plugin-wdm';
+
+import {registerPlugin} from '@ciscospark/spark-core';
+import Credentials from './credentials';
+import config from './config';
+
+registerPlugin(`credentials`, Credentials, {
+  config
+});
+
+export {
+  apiScope,
```

Comment:
I don't see apiScope defined in this file... Might just be that this is a set-up PR and it'll be in the next?